plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiMetadataProcessor.kt

```diff
@@ -163,6 +163,9 @@ private fun KmPackage.removePrivateDeclarations() {
 }
 
 private fun KmDeclarationContainer.removePrivateDeclarations() {
+    functions.sortBy(KmFunction::name)
+    properties.sortBy(KmProperty::name)
+
     functions.removeIf { it.visibility.isPrivate }
     properties.removeIf { it.visibility.isPrivate }
```

Comment:
```suggestion
    functions.removeIf { it.visibility.isPrivate }
    properties.removeIf { it.visibility.isPrivate }

    functions.sortBy(KmFunction::name)
    properties.sortBy(KmProperty::name)
```
Might as well sort after removal. Otherwise you're wasting time on sorting entries that are destined to be removed anyway.

Reply:
Thanks for catching it, noticed it after creating the PR but forgot to push.

Reply:
My dog ate it! I swear!